docs/en/ingest-management/commands.asciidoc

```diff
@@ -31,31 +31,90 @@ NOTE: You might need to log in as a root user to run these commands.
 [[elastic-agent-enroll-command]]
 == elastic-agent enroll
 
-Enroll the {elastic-agent} in {fleet}.
+Enroll the {agent} in {fleet}.
+
+Use this command to enroll the {agent} in {fleet} without installing the
+agent as a service. You will need to do this if you installed
+the {agent} from a DEB or RPM package and plan to use systemd commands to
+start and manage the service. This command is also useful for testing
+{agent} prior to installing it.
+
+If you've already installed {agent}, use this command to modify the settings that {agent} runs with. 
```

Comment:
@EricDavisX Edited my previous fix. I think I misunderstood your statement, which suggests that my change here might still be too vague. WDYT?

Reply:
That's actually perfect, its the exact purpose / use for 'enroll'. :)  no need to unenroll first - you just overwrite / change it and keep going.